code/gradle.properties

```diff
@@ -15,4 +15,6 @@ mavenRepoDescription=Adobe Experience Platform Edge Consent Collection extension
 mavenUploadDryRunFlag=false
 
 # production versions for production build
-mavenCoreVersion=1.8.0
\ No newline at end of file
+mavenCoreVersion=1.8.0
+
+android.disableAutomaticComponentCreation=true
```

Comment:
nit: move this up with the other "android" configuration options so they are grouped together.